torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -59,13 +59,21 @@ void replaceConvBiasWithGetAttr(Module& module) {
 
 void addBiasForConvIfNone(Module& module, const std::string& pattern_name) {
   auto t = module.type()->expect<ClassType>();
-  auto real_typename = t->name()->qualifiedName();
-  if (real_typename.size() >= pattern_name.size() &&
+  const std::string real_typename = t->name()->qualifiedName();
+
+  // checks if current module name ends with the right suffix
+  bool real_typename_ends_with_pattern_name =
+    real_typename.size() >= pattern_name.size() &&
       (0 ==
        real_typename.compare(
            real_typename.size() - pattern_name.size(),
            pattern_name.size(),
-           pattern_name))) {
+           pattern_name));
+  // we need to treat quantized modules in a special way
+  bool real_typename_contains_quantized =
```

Comment:
if there is a better way to check for this which wouldn't catch people naming their floating point modules ".*quantized.*", I'm flexible

Reply:
can we do this: https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/jit/passes/quantization/helper.cpp.html#_ZN5torch3jit16is_conv2d_moduleERKNS0_5MatchERKSt13unordered_mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPNS0_5ValueESt4hashISA_ESt8equal_11899141

Reply:
we can add bias only when the module is conv{1,2,3}d